cw-orch/src/osmosis_test_tube/core.rs

```diff
@@ -338,3 +339,16 @@ impl Stargate for OsmosisTestTube {
         })
     }
 }
+
+impl WasmCodeQuerier for OsmosisTestTube {
```

Comment:
Do you know if there's a way to implement these functions for test-tube @Kayanski?

Reply:
yes ! With raw queries :)

Reply:
It works well now :)